README.md

```diff
@@ -39,6 +39,7 @@
       platform [command]   update a platform version
       help [command]       output usage information
       version              output version number
+      analytics            turn analytics on or off, or view current status
```

Comment:
This is 76 char, and I think clearer than your suggestion. 
Your suggestion saves 3 char.